pydis_site/apps/content/resources/guides/python-guides/bot_presence.md

```diff
@@ -0,0 +1,181 @@
+        status=discord.Status.idle,
+    )
+    ```
+
+- Playing activity
+    ```py
+    import discord
+    from discord.ext import commands
+
+    bot = commands.Bot(
+    command_prefix="<Your prefix>",
+    status=discord.Status.online,
+    activity=discord.Game(name="with wumpus"),
+    )
+    ```
+
+- Streaming activity
+    ```py
+    import discord
+    from discord.ext import commands
+
+    bot = commands.Bot(
+        command_prefix="<Your prefix>",
+        status=discord.Status.online,
+        activity=discord.Streaming(name="Wumpus Stream", url="the streams url"),
+    )
+    ```
+
+- Listening activity
+    ```py
+    import discord
+    from discord.ext import commands
+
+    bot = commands.Bot(
+        command_prefix="<Your prefix>",
+        status=discord.Status.online,
+        activity=discord.Activity(
+            type=discord.ActivityType.listening, name="The wumpus  song"
+        ),
+    )
+    ```
+
+- Watching activity
+    ```py
+    import discord
+    from discord.ext import commands
+
+    bot = commands.Bot(
+        command_prefix="<Your prefix>",
+        status=discord.Status.online,
+        activity=discord.Activity(
+            type=discord.ActivityType.watching, name="The wumpus movie"
+        ),
+    )
+    ```
+
+- Competing activity
+    ```py
+    import discord
+    from discord.ext import commands
+
+    bot = commands.Bot(
+        command_prefix="<Your prefix>",
+        status=discord.Status.online,
+        activity=discord.Activity(
+            type=discord.ActivityType.competing, name="The wumpus competition"
+        ),
+    )
+    ```
+
+## Changing presence after logging in
+You can also use the [`change_presence()`](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html?highlight=bot#discord.ext.commands.Bot.change_presence) method.
+
+> **_NOTE:_** The method should not be used inside of **on_ready()** as this event can be triggered multiple times after logging in. Resulting in sending multiple of the same `PRESENCE` gateway event payload to the gateway.
+
+## Changing activity after a certain period of time
+A large number of bots have presences that change by themselves after a certain period. In discord.py this can be done using a task loop. You can read more about tasks [in the documentation](https://discordpy.readthedocs.io/en/latest/ext/tasks/index.html).
+
+```py
+import discord
+from discord.ext import commands, tasks
+import random
+
+bot = commands.Bot(command_prefix="<Your prefix>")
+
+bot.changeable_activites = (
+    "Call of Duty: Warzone",
+    "Fall Guys",
+    "Fortnite",
+    "Call of Duty: Black Ops IV",
+    "Sea of Thieves",
+    "League of Legends",
+    "Valorant",
+)
+
+@tasks.loop(minutes=2)
+async def change_activity():
+    await bot.wait_until_ready()
+    new_activity = random.choice(bot.changeable_activites)
+    await bot.change_presence(activity=discord.Game(new_activity))
+
+change_activity.start()
+```
+
```

Comment:
Could we also add one section for changing presence on command? I see people trying this a lot and having a quick snippet would be useful